### PR TITLE
impr: Refactor order related e2e to not run tests in a loop

### PR DESCRIPTION
**Pull request by tomaszpowroznik**

### CODE REVIEW

File: e2e/fixtures/orders/dischargeOfCareOrder.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      year: '2020',
+    },
+    judgeAndLegalAdvisor: {
+      judgeTitle: 'Her Honour Judge',
+      judgeLastName: 'Judy',
+      judgeEmailAddress: 'test@test.com',
+      legalAdvisorName: 'Fred Frederickson',
+    },
+    careOrders: [0],
+    closeCase: false,
```
Comment (akomisarek) [inline e2e/fixtures/orders/dischargeOfCareOrder.js:18]: Is closing a case relevant for discharge? is it final order?

File: e2e/fixtures/orders/finalCareOrder.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      month: '01',
+      year: '2020',
+    },
+    judgeAndLegalAdvisor: {
+      judgeTitle: 'Her Honour Judge',
+      judgeLastName: 'Judy',
+      judgeEmailAddress: 'test@test.com',
+      legalAdvisorName: 'Fred Frederickson',
+    },
+    closeCase: false,
```
Comment (akomisarek) [inline e2e/fixtures/orders/finalCareOrder.js:18]: Are we testing in any flow closeCase true?

File: e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js [hunk 3]
```diff
@@ -22,13 +28,38 @@ BeforeSuite(async (I, caseViewPage, submitApplicationEventPage, enterFamilyManCa
...
+
+Scenario('Judiciary creates final supervision order', async (I, caseViewPage, createOrderEventPage) => {
+  await verifyOrderCreation(I, caseViewPage, createOrderEventPage, finalSuperVisionOrder);
+});
+
+Scenario('Judiciary creates emergency protection order', async (I, caseViewPage, createOrderEventPage) => {
+  await verifyOrderCreation(I, caseViewPage, createOrderEventPage, emergencyProtectionOrder);
+});
+
+Scenario('Judiciary creates interim care order', async (I, caseViewPage, createOrderEventPage) => {
```
Comment (akomisarek) [inline e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js:47]: Are you going to delete those checks? You mentioned you will like to delete them.. Ah in separate PR ok!

File: e2e/fixtures/orders/finalCareOrder.js (hunk 2 above)
Comment (lewisbirks) [reply e2e/fixtures/orders/finalCareOrder.js:18]: We want to test closing the case through the close case event so we don't test the close case functionality from an order in the e2e, maybe that is an oversight on our part

**Pull request merged**